app/models/forms/integrity_warnings/question_warner.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Forms
+  module IntegrityWarnings
+    # Enumerates integrity warnings for QingGroups
+    class QuestionWarner < Warner
+      protected
+
+      # See Warner#warnings for more info on the expected return value here.
+      def careful_with_changes
+        [{in_use: :form_list}, :published, :standard_copy]
+      end
+
+      def features_disabled
+        # We include standard_copy here because editing the question code is not allowed
+        # for standard copies.
+        %i[published data standard_copy]
+      end
+
+      def form_list
+        form_count = object.forms.size
+        more_suffix =
+          if form_count > MAX_FORMS
+            str = I18n.t("integrity_warnings.more_suffix", count: form_count - 3)
+            "(+#{str})"
```

Comment:
Note "3 more" might become "more 3" in other languages, so `+` at beginning might be better placed in the translation itself. Probably nbd though.